StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Elements/LinkCardEditElement.swift

```diff
@@ -111,14 +125,21 @@ final class LinkCardEditElement: Element {
         return panElement
     }()
 
-    private lazy var cvcElement = TextFieldElement(
-        configuration: TextFieldElement.CVCConfiguration(
-            cardBrandProvider: { [weak self] in
-                self?.paymentMethod.cardDetails?.stpBrand ?? .unknown
-            }
-        ),
-        theme: theme
-    )
+    private lazy var cvcElement: TextFieldElement = {
+        let cvcElement = TextFieldElement(
+            configuration: TextFieldElement.CVCConfiguration(
+                // If we're using a placeholder, we just fill up the view with zeros.
```

Comment:
We'll render it as "•••" though, right?

Reply:
Yep, but with this ugly workaround 😕 I just noticed that `CensoredCVCConfiguration` exists, which gives us that functionality for free, but it makes the look different from the other disabled fields. What do you think about this?

![Simulator Screenshot - iPhone 16 Pro - 2025-03-28 at 11 26 10](https://github.com/user-attachments/assets/8195d131-8ba2-4e90-920d-afb27fdb574b)